assets/app/styles/_core.less

```diff
@@ -203,6 +203,25 @@ html, body {
   }
 }
 
+.nav-tabs {
+  font-size: @font-size-base;
+  a {
+    // ui.bootstrap.tabs doesn't set the href attr, so the cursor style is
+    // required or it's a vertical bar
+    cursor: pointer;
+  }
+  li.active {
+    a, a:hover {
+      // Remove when we update patternfly
+      background-color: #fff;
```

Comment:
@sg00dwin FYI, we'll want to remove this if the newer patternfly uses this style for active tabs.